source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,250 @@
+- :java-core-api:`TextSearchOptions <com/mongodb/client/model/TextSearchOptions.html>`
+
+.. _term_search:
+
+Search Text by a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass the term as a string to the ``Filters.text()`` method to specify a
+term in your text search. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "Fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin termExample
+   :end-before: end termExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] }
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+
+If you want to match multiple terms in your text search, separate them
+with spaces in the ``Filters.text()`` builder method. When you pass the 
+text search query returned by the builder method to the
+``find()`` method, it returns documents that match any of the terms. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+terms "Fate" or "7".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin multipleTermExample
+   :end-before: end multipleTermExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass a phrase with escaped quotes to the ``Filters.text()`` method to
+specify the phrase in your text search. If you don't add escaped quotes,
+the method runs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+phrase "Fate of the Furious". 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+ 
+Search Text with Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass a term prefixed with a minus sign ``-`` to the ``Filters.text()``
+method to specify each term you want to exclude in your text search.
+
+You must have at least one term to include to specify terms to exclude.
+If you have multiple text indexes, the search applies to all indexes.
```

Comment:
A couple items here:
1) Does this sentence have anything to do with excluding terms? If you're addressing [this comment](https://github.com/mongodb/docs-java/pull/85/files#r646784453), I was looking for info on whether the exclusion would apply to all indexed fields.
2) A text search doesn't "[apply] to indexes". Instead, a search can use indexed data to retrieve results. Instead of the current sentence, consider the following:

```suggestion
If your collection has more than one text field indexed, the search and exclusion applies to all the indexed fields.
```

Reply:
Removed this section.